src/kernel/CMakeLists.txt

```diff
@@ -98,6 +98,39 @@ set_target_properties(bitcoinkernel PROPERTIES
   CXX_VISIBILITY_PRESET default
 )
 
+# When building the static library, install all static libraries the
+# bitcoinkernel depends on.
+if (NOT BUILD_SHARED_LIBS)
```

Comment:
style nit: All other CMake code uses the `if()` command without an additional space.